parsing/g_proofs.ml4

```diff
@@ -19,6 +19,25 @@ let thm_token = G_vernac.thm_token
 
 let hint = Gram.entry_create "hint"
 
+let warn_deprecated_focus =
+  CWarnings.create ~name:"deprecated-focus" ~category:"deprecated"
+         (fun () ->
+           Pp.strbrk
+             "Focus is deprecated; use bullets or focusing brackets instead"
+         )
+
+let warn_deprecated_focus_n n =
+  CWarnings.create ~name:"deprecated-focus" ~category:"deprecated"
+         (fun () ->
+           Pp.strbrk ( "Focus " ^ string_of_int n ^ " is deprecated; use "
```

Comment:
I think it would make more sense to say "The Focus command is deprecated". It is not really `Focus 2` or `Focus 3` that is deprecated...

Reply:
Right, but do you agree that it is a good idea to propose an alternative which reuses the number from the `Focus` command?

Reply:
Yes.

Reply:
Changed.